x-pack/plugins/cloud/server/config.ts

```diff
@@ -28,28 +28,36 @@ const fullStoryConfigSchema = schema.object({
   ),
 });
 
+const chatConfigSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: false }),
+  chatURL: schema.maybe(schema.string()),
+});
+
 const configSchema = schema.object({
-  id: schema.maybe(schema.string()),
   apm: schema.maybe(apmConfigSchema),
-  cname: schema.maybe(schema.string()),
   base_url: schema.maybe(schema.string()),
-  profile_url: schema.maybe(schema.string()),
+  chat: chatConfigSchema,
+  chatIdentitySecret: schema.maybe(schema.string()),
```

Comment:
Just to be sure, `chatIdentitySecret` is outside of the `chat` config to avoid exposing it to the client, right?

Reply:
That's right.